kube-runtime/src/lib.rs

```diff
@@ -6,12 +6,13 @@
 #![allow(clippy::default_trait_access)]
 
 pub mod controller;
+mod manager;
```

Comment:
Seems like `manager` should be a submodule of `controller`, if it's only exposed via that module?

Reply:
i'll move it into the controller module.